symplyphysics/laws/conservation/charge_is_constant.py

```diff
@@ -0,0 +1,37 @@
+from sympy import Eq, dsolve, Derivative
+from symplyphysics import (
+    Symbol,
+    Function,
+    units,
+    print_expression,
+    Quantity,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The total charge of any isolated system is conserved.
+
+# Law: dq/dt = 0
+## q - total charge of the system
+## d/dt - derivative with respect to time
+
+# Conditions
+## - The system is isolated, i.e. no particles can leave it
+
+total_charge = Function("total_charge", units.charge)
+time = Symbol("time", units.time)
+
+law = Eq(Derivative(total_charge(time), time), 0)
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(total_charge_before_=total_charge)
+@validate_output(total_charge)
+def calculate_charge_after(total_charge_before_: Quantity) -> Quantity:
+    dsolved = dsolve(law, total_charge(time))
+    result_charge = dsolved.subs("C1", total_charge_before_).rhs
```

Comment:
Here you should know, what's on the left side of the result of dsolve(). I suggest not to rely on it. Instead you can take the dsolved, which tells that charge equals to `C1`. And solve a set of 2 equations: charge_before = C1, charge_after = C1. Solution will tell that charge_after = charge_before.

Reply:
I see, it's just that I sometimes use already existing laws and they are written that way (namely, mass_is_constant and mechanical_energy_is_constant are the ones I referenced). I see the advantage of your suggestion, though. I just want to say that it would probably be easier if there were some conventions throughout the whole project which developers would follow.

Reply:
We cannot cover every line of code with conventions. It is better to add TODO hint everywhere you see the same problem.

Reply:
Okay, I see